utils/map.js

```diff
@@ -18,10 +18,11 @@ export function coordsToExtent(coords) {
 
 export function newMap(element, mapOptions) {
   // lazy-load the map modules
-  return loadModules(['esri/Map', 'esri/views/MapView', 'esri/Graphic'], {
-    // TODO: also lazy-load the CSS
-    // css: 'https://js.arcgis.com/4.11/esri/css/main.css'
-  }).then(([Map, MapView, Graphic]) => {
+  return loadModules(
+    ['esri/Map', 'esri/views/MapView', 'esri/Graphic'],
+    // NOTE: keep this current w/ the latest version of the JSAPI
+    { css: 'https://js.arcgis.com/4.11/esri/css/main.css' }
```

Comment:
https://github.com/tomwayson/create-arcgis-app/pull/35#discussion_r280074678

Reply:
Cool feature, good to know. Updated.